pkg/cmd/condition/describe.go

```diff
@@ -0,0 +1,183 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package condition
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+	"text/template"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/formatted"
+	"github.com/tektoncd/cli/pkg/printer"
+	"github.com/tektoncd/cli/pkg/validate"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+const templ = `{{decorate "bold" "Name"}}:	{{ .Condition.Name }}
+{{decorate "bold" "Namespace"}}:	{{ .Condition.Namespace }}
+{{- if ne .Condition.Spec.Description "" }}
+{{decorate "bold" "Description"}}:	{{ .Condition.Spec.Description }}
+{{- end }}
+
+{{ decorate "check" ""}}{{decorate "underline bold" "Check\n"}}
+ {{decorate "bold" "Name"}}:	{{ .Condition.Spec.Check.Name }}
+ {{decorate "bold" "Image"}}:	{{ .Condition.Spec.Check.Image }}
+ {{- if .Condition.Spec.Check.Command }}
+ {{decorate "bold" "Command"}}:	{{ .Condition.Spec.Check.Command }}
+ {{- end }}
+ {{- if .Condition.Spec.Check.Args }}
+ {{decorate "bold" "Args"}}:	{{ .Condition.Spec.Check.Args }}
+ {{- end }}
+ {{- if ne .Condition.Spec.Check.Script "" }}
+ {{decorate "bold" "Script"}}:	{{ .Condition.Spec.Check.Script }}
+ {{- end }}
+
+{{decorate "resources" ""}}{{decorate "underline bold" "Resources\n"}}
+{{- $rl := len .Condition.Spec.Resources }}{{ if eq $rl 0 }}
+ No resources
+{{- else }}
+ NAME	TYPE
+{{- range $r := .Condition.Spec.Resources }}
+ {{decorate "bullet" $r.Name }}	{{ $r.Type }}
+{{- end }}
+{{- end }}
+
+{{decorate "params" ""}}{{decorate "underline bold" "Params\n"}}
+{{- $l := len .Condition.Spec.Params }}{{ if eq $l 0 }}
+ No params
+{{- else }}
+ NAME	TYPE	DESCRIPTION	DEFAULT VALUE
+{{- range $p := .Condition.Spec.Params }}
+{{- if not $p.Default }}
+ {{decorate "bullet" $p.Name }}	{{ $p.Type }}	{{ formatDesc $p.Description }}	{{ "---" }}
+{{- else }}
+{{- if eq $p.Type "string" }}
+ {{decorate "bullet" $p.Name }}	{{ $p.Type }}	{{ formatDesc $p.Description }}	{{ $p.Default.StringVal }}
+{{- else }}
+ {{decorate "bullet" $p.Name }}	{{ $p.Type }}	{{ formatDesc $p.Description }}	{{ $p.Default.ArrayVal }}
+{{- end }}
+{{- end }}
+{{- end }}
+{{- end }}
+`
+
+func describeCommand(p cli.Params) *cobra.Command {
+	f := cliopts.NewPrintFlags("describe")
+	eg := `Describe a Condition of name 'foo' in namespace 'bar':
+
+    tkn condition describe foo -n bar
+
+or
+
+    tkn cond desc foo -n bar
+`
+
+	c := &cobra.Command{
+		Use:          "describe",
+		Aliases:      []string{"desc"},
+		Short:        "Describes a condition in a namespace",
```

Comment:
```suggestion
		Short:        "Describes condition in a namespace",
```

Reply:
Perhaps `Describe Conditions in a namespace`

Reply:
Fixed